docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -150,6 +150,79 @@ To investigate further, you can view the details of the build captured as labels
 .Contextual attributes of a Jenkins pipeline execution in Elastic Observability
 image::images/jenkins-pipeline-context.png[Attributes of a Jenkins pipeline execution]
 
+[discrete]
+[[ci-cd-store-jenkins-logs]]
+== Store Jenkins pipeline logs in Elastic
+
+Jenkins pipeline build logs can be sent through OpenTelemetry Protocol (OTLP) to be stored
+in an observability backend alongside the traces of the pipeline builds and Jenkins health metrics.
+Storing logs in an observability backend provides several benefits including:
+
+* Better observability, monitoring, alerting, and troubleshooting of the Jenkins instance
+due to the unification of all the signals in the observability backend.
+* Better traceability and control of long term storage for builds so you can more easily
+audit the Software Delivery Lifecycle.
+* Better scalability and reliability of Jenkins by greatly reducing the quantity of data stored in
+Jenkins and limiting the well known file system performance of Jenkins when storing a large history of builds.
+
+[discrete]
+[[ci-cd-store-jenkins-logs-requirements-configuration]]
+=== Requirements and configuration
+
+Storing Jenkins pipeline logs in Elastic requires:
+
+* Elastic Observability version 8.1 or higher.
+* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry integrationmust be
+reachable from the Jenkins Agents (i.e. don't specify a localhost OTLP endpoint unless OpenTelemetry
+collectors are also deployed on the Jenkins Agents).
+* When using OpenTelemetry Collectors, set up a logs pipeline in addition to the traces and metrics pipelines.
+
+To store pipeline logs in Elastic:
+
+1. Navigate to the _OpenTelemetry_ section of Jenkins configuration screen.
+2. Set the _OTLP Endpoint_.
+3. Use the _Add Visualisation Observability Backend_ dropdown to select the *Elastic Observability* option.
+4. Set the _Kibana base URL_.
+5. Click the _Advanced_ button to choose a storage integration strategy.
+There are two options for storing pipeline logs in Elastic Observability:
+    * *Store pipeline logs in Elastic and visualize logs exclusively in Elastic*,
+    which means logs will no longer be visible through the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana>> below.
+    * *Store pipeline logs in Elastic and visualize logs both in Elastic and through Jenkins*,
+    which means you can view logs stored in Elastic on demand in the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana-and-jenkins>> below.
+
```

Comment:
```suggestion

Visualizing logs both in Elastic and through Jenkins is recommended with an Elasticsearch access can be enable from the Jenkins Controller. It provides continuity in the user experience continuing to render the logs in the Jenkins GUI and it enables verification of the Elasticsearch setup.
```

Reply:
>Visualizing logs both in Elastic and through Jenkins is recommended with an Elasticsearch access can be enable from the Jenkins Controller.

What are you trying to communicate in this sentence? Could it just be:

>Visualizing logs both in Elastic and through Jenkins is recommended.

Reply:
I attempted to make this clearer in https://github.com/elastic/observability-docs/pull/1688/commits/de6024a25bd751624d0cd55bce01bffab9c7ce32.

Reply:
I like very much https://github.com/elastic/observability-docs/commit/de6024a25bd751624d0cd55bce01bffab9c7ce32 , is it merged in this PR?

Reply:
Yes! https://github.com/elastic/observability-docs/pull/1688/files#diff-75b8ab84b7edfb289a7f2e7e392dd0b801189e321a0d66ac4c660befe5d4178bR196-R197